corehq/apps/reports/standard/__init__.py

```diff
@@ -46,9 +50,24 @@ def default_report_url(self):
     @property
     def template_context(self):
         context = super().template_context
+
+        email_form = EmailReportForm()
+        web_user_emails = (
+            row['doc'].get('email') or row['doc'].get('username')
+            for row in get_all_user_rows(
```

Comment:
Apologies for not bringing it up earlier, but I wonder if doing an ES query would be much performant here?